src/layouts/webapp/Messages/ChatList.tsx

```
@@ -73,7 +90,30 @@ export default function ChatList() {
   const dispatch = useAppDispatch();
 
-  // const [qString, setQString] = useState('');
+  const [qString, setQString] = useState('');
+
+  const debouncedSearch = useDebounce(qString, 300);
+
+  const filteredConversations = useMemo(() => {
+    const normalizeText = (text: string) =>
+      text
+        .normalize('NFD')
+        .replace(/[\u0300-\u036f]/g, '')
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)